core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -479,6 +497,16 @@ private[spark] class MapOutputTrackerMaster(
     }
   }
 
+  def updateMapOutput(shuffleId: Int, mapId: Long, bmAddress: BlockManagerId): Unit = {
+    shuffleStatuses.get(shuffleId) match {
+      case Some(shuffleStatus) =>
+        shuffleStatus.updateMapOutput(mapId, bmAddress)
+        shuffleStatus.invalidateSerializedMapOutputStatusCache()
```

Comment:
`ShuffleStatus.updateMapOutput` calls `invalidateSerializedMapOutputStatusCache`, do we need calling it again here?

Reply:
Good catch, I'll drop it.